xla/xla_data.proto

```diff
@@ -82,6 +82,18 @@ enum PrimitiveType {
   F8E4M3FN = 20;
   F8E4M3B11FNUZ = 23;
 
+  // FP8 dtypes, as described in this paper: https://arxiv.org/abs/2206.02915
+  //
+  // F8E5M2FNUZ has 5 exponent bits and 2 mantissa bits.
+  // F8E4M3FNUZ has 4 exponent bits and 3 mantissa bits.
+  //
+  // The "FNUZ" means only Finite and NaN values are supported; zero is
+  // unsigned. Unlike IEEE types, infinities are not supported.  NaN is
+  // represented when the exponent and mantissa bits are all 0s with a sign bit
+  // of 1. All other values are finite.
```

Comment:
I think it's worth mentioning the exponent biases, since they're one lower than F8E5M2 and F8E4M3FN.

Reply:
I've taken the description from the [StableHLO RFC](https://github.com/openxla/stablehlo/blob/aac89de63ba020d26b7afad7af6b127e7184890c/rfcs/20230321-fp8_fnuz.md?plain=1#L24-L27).